misc/python/materialize/output_consistency/execution/sql_executor.py

```diff
@@ -31,6 +31,12 @@ def __init__(self, message: str):
 class SqlExecutor:
     """Base class of `PgWireDatabaseSqlExecutor` and `DryRunSqlExecutor`"""
 
+    def __init__(
```

Comment:
I understand what you mean but am reluctant to apply it here since the class as such is not data class at all. Though, I will consider it in other places in future.